types/jquery/JQueryStatic.d.ts

```diff
@@ -1611,6 +1611,21 @@ $.get( "test.php" );
 ```
      */
     get(url_settings?: string | JQuery.UrlAjaxSettings): JQuery.jqXHR;
+    /**
+     * Load data from the server using a HTTP GET request.
+     * @param url A string containing the URL to which the request is sent.
+     * @param data A plain object or string that is sent to the server with the request.
+     * @param dataType The type of data expected from the server. Default: Intelligent Guess (xml, json, script, text, html).
+     * @see \`{@link https://api.jquery.com/jQuery.get/ }\`
+     * @since 1.0
+     * @example ​ ````Request the test.php page, send an object with data, receive a json response.
+```javascript
+$.get( "test.php", { name: 'John' }, 'json' );
+```
+     */
+    get(url: string,
```

Comment:
remove your definition and update this [get](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cd0e972f17bf5c892799692760828abe615430ee/types/jquery/JQueryStatic.d.ts#L1559-L1561) to 
```
get(url: string,
        data_success: JQuery.PlainObject | string | JQuery.jqXHR.DoneCallback | null,
        dataType: string): JQuery.jqXHR;
```